src/NServiceBus.TransactionalSession/ConsumeMessageException.cs

```diff
@@ -2,6 +2,7 @@
 {
     using System;
 
+    [Serializable]
```

Comment:
why is that needed?

Reply:
technically not needed. I fell into the trap of starting to go down the path of the exception best practices. Happy to remove again

Reply:
Removed